components/ILIAS/Authentication/classes/class.ilSessionControl.php

```diff
@@ -27,10 +27,7 @@ class ilSessionControl
      * default value for settings that have not
      * been defined in setup or administration yet
      */
-    public const DEFAULT_MAX_COUNT = 0;
     public const DEFAULT_MIN_IDLE = 15;
```

Comment:
Do we still need the "Min Idle Time" setting?

Reply:
it's still being used in ilSCORM13PlayerGUI.

I could move that constant into the scorm class directly or replace it with hard coded values though.